tests/e2e/e2e_test.go

```diff
@@ -1,15 +1,21 @@
-//go:build e2e
```

Comment:
hey why you delete this ? Test fail because you remove this line

Reply:
Whoops! This was from when I was trying to get it to run locally. Reverting back to how it was before sorry!